app/src/main/java/com/bpn/trainme/presentation/features/splash/SplashViewModel.kt

```diff
@@ -13,13 +13,13 @@ class SplashViewModel @Inject constructor(
 
     init {
         updateState(UiState.Idle)
-        startSplashTimer()
+//        startSplashTimer()
     }
 
-    private fun startSplashTimer() {
-        launchInViewModelScope {
-            delay(2000)
-            emitEvent(SplashUiEvent.NavigateToLogin)
-        }
-    }
+//    private fun startSplashTimer() {
```

Comment:
Perhaps you can remove this altogether, as you can always find it in the git history